tests/test_update.py

```diff
@@ -17,6 +26,95 @@
     clone_package,
 )
 
+@record_requests_module
+@record_git_module
+@record_tempfile_module
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="packit.utils.run_command_remote", decorate=Simple.decorator_plain()
+    )
+)
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="packit.fedpkg.FedPKG.clone",
+        decorate=StoreFiles.where_arg_references(
+            key_position_params_dict={"target_path": 2}
+        ),
+    )
+)
+# Be aware that decorator stores login and token to test_data, replace it by some value.
+# Default precommit hook doesn't do that for copr.v3.helpers, see README.md
```

Comment:
Looks like some optional `replace`/`cleanup` function might come handy. WDYT, @jscotka ? Don't we have already an issue for this?

Reply:
It is easy, described in doc for packit, some of them are part of default precommit config  inside requre, this one is not part of that. so that it can be added, we should create an issue to requre with mentioned replacement:
https://github.com/packit/packit/blob/master/tests_recording/README.md#postprocessing
```
requre-patch purge --replaces "copr.v3.helpers:login:str:somelogin" --replaces "copr.v3.helpers:token:str:sometoken" tests_recording/test_data/*/*yaml
```
into https://github.com/packit/requre/blob/master/.pre-commit-hooks.yaml file

seems like very easy bug, good fist issue

Reply:
No, I meant something like this:

```python
@apply_decorator_to_all_methods(
    replace_module_match(
        what="packit.utils.run_command_remote",
        decorate=Simple.decorator_plain(),
        cleanup=lambda x: x.replace("a", "b"),
    )
)
```

Done on write-mode -- let's discuss in a new require issue: https://github.com/packit/requre/issues/137